Sources/Leaf/Tag/Loop.swift

```diff
@@ -6,52 +6,100 @@ public final class Loop: LeafTag {
     public func render(parsed: ParsedTag, context: LeafContext, renderer: LeafRenderer) throws -> Future<LeafData?> {
```

Comment:
This function has gotten too big. We need to break it out into separate, private funcs.